src/test/java/com/marizueva/laboratory/hw5/utils/UsedInTestTerms.java

```diff
@@ -0,0 +1,15 @@
+package com.marizueva.laboratory.hw5.utils;
+
+public class UsedInTestTerms {
```

Comment:
should be `final` and has `private` constructor

Reply:
fixed